.github/project-labeler.yml

```diff
@@ -109,6 +109,9 @@ pie-textarea:
 pie-toast:
 - 'packages/components/pie-toast/**/*'
 
+pie-radio:
```

Comment:
Nit: alphabetical order please. For this file, the CI tokens and also pie-webc's package.json